assets/javascripts/kitten/components/cards/reward-card/image.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component, Fragment } from 'react'
+import Radium from 'radium'
+import PropTypes from 'prop-types'
+import { styles } from 'kitten/components/cards/reward-card'
+
+const RewardCardImageBase = ({ isDisabled, imageProps }) => {
+  const imageStyles = [isDisabled && styles.disabled]
+  const shouldDisplayImage = imageProps && imageProps.src
+
+  if (!shouldDisplayImage) return null
+
+  return (
+    <div style={imageStyles} disabled={isDisabled}>
+      <img {...imageProps} style={styles.image} />
```

Comment:
Que penses-tu d'isoler le `alt` de `imageProps` afin de l'ajouter quand il n'y en a pas ?

Reply:
Oui, c'est une très bonne idée.

Reply:
Un truc dans ce style : 

```suggestion
      <img {...imageProps} alt={ imageProps.alt || '' } style={styles.image} />
```